algorithms/kernel/cholesky/cholesky_impl.i

```diff
@@ -89,17 +89,16 @@ Status CholeskyKernel<algorithmFPType, method, cpu>::copyMatrix(NumericTableIfac
 {
     if(isFull<algorithmFPType, cpu>(rLayout))
     {
-        if(!copyToFullMatrix(iLayout, pA, pL, dim))
+        if (!copyToFullMatrix(iLayout, pA, pL, dim))
             return Status(ErrorIncorrectTypeOfInputNumericTable);
+        else
+            return Status();
 
     }
     else
     {
-        if(!copyToLowerTrianglePacked(iLayout, pA, pL, dim))
-            return Status(ErrorIncorrectTypeOfOutputNumericTable);
-
+        return copyToLowerTrianglePacked(iLayout, pA, pL, dim);
```

Comment:
why did we change only return value only for copyToLowerTrianglePacked?
But not for copyToFullMatrix

Reply:
because for copyToLowerTrianglePacked status is might have 3 different options, for copyToFullMatrix that options only 2 and bool type is enough.